eta/core/models.py

```diff
@@ -1383,17 +1405,11 @@ def parse_name(name):
         Returns:
             base_name: the base name of the model
             version: the version of the model, or None if no version was found
-
-        Raises:
-            ModelError: if the model name was invalid
         """
-        chunks = name.split("@")
+        chunks = name.rsplit("@", 1)
         if len(chunks) == 1:
             return name, None
 
-        if chunks[1] == "" or len(chunks) > 2:
```

Comment:
What happens now then in the case that `version==""`? I guess it would be like `ab@`.
Another case of different behavior is `a@b@1.2`.

Reply:
The `@` feature is undocumented and unused in FO to-date, this versioning feature is mostly leftover cruft from 2017. It does work though!

Reply:
ok